README.md

```diff
@@ -1,55 +1,70 @@
-# LitmusKt 
+# LitmusKt
 
 **LitmusKt** is a litmus testing tool for Kotlin.
-Litmus tests are small concurrent programs exposing various relaxed behaviors,
-arising due to compiler or hardware optimizations (for example, instruction reordering).
+Litmus tests are small concurrent programs exposing various relaxed behaviors, arising due to compiler or hardware
+optimizations (for example, instruction reordering).
 
-This project is in an **experimental** stage of the development. 
+This project is in an **experimental** stage of the development.
 The tool's API is unstable and might be a subject to a further change.
 
 ## Setup
 
 Simply clone the project and run `./gradlew build`.
 
-### Running
+Note that for Kotlin/JVM this project relies on [jcstress](https://github.com/openjdk/jcstress), so you also need to
+set `JCS_DIR` environment variable to `/path/to/jcstress/folder`.
 
-All classic Gradle building tasks (like `run`, `build`, `-debug-` or `-release-`) work as expected. 
-The only important distinction is that on various platforms these tasks are named differently.
+Tip: if you wish to run Gradle tasks from IDEA, you need to set `JCS_DIR` for it too, and a handy way to do that is to
+run IDEA from bash: `idea & disown -a` (note the singular `&`).
```

Comment:
Looks cryptic.
Is there any other way to propagate this `JCS_DIR` parameter?

Reply:
Unfortunately, I don't think so. If you want to run gradle tasks from console, then it's as simple as setting one env var. If you want to use IDEA, then this variable is necessary even for project sync, and it will always fail unless the entire IDEA sees this variable.

Reply:
https://www.youtube.com/watch?v=PF8d1r-jTN8